packages/block-library/src/post-comments-count/edit.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useEntityId } from '@wordpress/core-data';
+import { useState, useEffect } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+
+function PostCommentsCountDisplay( { className } ) {
+	const postId = useEntityId( 'postType', 'post' );
+	const [ commentsCount, setCommentsCount ] = useState();
+	useEffect( () => {
+		const currentPostId = postId;
+		apiFetch( {
+			path: addQueryArgs( '/wp/v2/comments', {
+				post: postId,
+			} ),
+			parse: false,
+		} ).then( ( res ) => {
+			// Stale requests will have the `currentPostId` of an older closure.
+			if ( currentPostId === postId ) {
+				setCommentsCount( res.headers.get( 'X-WP-Total' ) );
+			}
+		} );
+	}, [ postId ] );
+	return (
+		<span
+			className={ classnames(
+				'wp-block-post-comments-count',
```

Comment:
This one is already generated and included in className prop I believe.

Reply:
You are right. Updated!